src/types/UiaTextRangeBase.hpp

```diff
@@ -135,9 +135,9 @@ namespace Microsoft::Console::Types
         // measure units in the form [_start, _end).
         // These are in the TextBuffer coordinate space.
         // NOTE: _start is inclusive, but _end is exclusive
-        COORD _start{};
-        COORD _end{};
-        bool _blockRange;
+        COORD _start{ 0, 0 };
+        COORD _end{ 0, 0 };

```

Comment:
No particular reason these are initialized to `{0,0}`. We should never actually hit this because, as mentioned in the PR description, all of the `RuntimeClassInitialize` set these to a value. I'm fine leaving this to `{}` but wanted to know what your thoughts are.

Reply:
I prefer `{}` because it signals to me that a (if the given type is a primitive type) specifically a `memset(this, 0, sizeof(*this))` is being done here. Others might have other opinions though...

Reply:
I prefer the `{}` as well here.